docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+
+services:
+    node:
+        build:
+            context: ./
+            dockerfile: ./Dockerfile
```

Comment:
If you don't have a custom name docker file, do we need to specify it in the "dockerfile" section ? 
It think it should automatically pick it up based on the 'context'

Reply:
Probably an artifact from https://gerrit.wikimedia.org/r/plugins/gitiles/wikibase/termbox/+/89dd6e5/docker-compose.yml#7

Reply:
It is indeed not needed. Removed it!